Gemfile

```diff
@@ -2,7 +2,7 @@
 source 'https://rubygems.org'
 
 group :development do
-  gem 'rspec', '~> 3.0'
+  gem 'rspec', '~> 3.5'
```

Comment:
Ok. So when rebasing, do I then create a new PR? since that changes the history of the branch. (I'm not that familiar with rebase.)